node/network/approval-distribution/src/lib.rs

```diff
@@ -1662,20 +1747,27 @@ impl ApprovalDistribution {
 
 	async fn run<Context>(self, ctx: Context) {
 		let mut state = State::default();
+		let mut reputation = ReputationAggregator::new();
+		let reputation_interval = std::time::Duration::from_millis(5);
```

Comment:
This is very low, likely no aggregation will actually happen, except maybe under load. I was thinking something like 30s to start with. We should do some Versi burn-ins to gauge the effectiveness of the aggregation.